pkgs/development/compilers/llvm/3.9/libc++abi.nix

```diff
@@ -5,7 +5,7 @@ stdenv.mkDerivation {
 
   src = fetch "libcxxabi" "1qi9q06zanqm8awzq83810avmvi52air6gr9zfip8mbg5viqn3cj";
 
-  buildInputs = [ cmake ] ++ stdenv.lib.optional (!stdenv.isDarwin && !stdenv.isFreeBSD) libunwind;
+  nativeBuildInputs = [ cmake ] ++ stdenv.lib.optional (!stdenv.isDarwin && !stdenv.isFreeBSD) libunwind;
```

Comment:
Please leave `libunwind` in the `buildInputs`.

Reply:
ok